learn/advanced/chronological_filtering_sorting.md

```diff
@@ -0,0 +1,116 @@
+# Filtering and sorting by date
+
+In this guide, you will learn about Meilisearch's approach to date and time values, how to prepare your dataset for indexing, and how to chronologically sort and filter search results.
+
+## Preparing your documents
+
+To filter and sort search results based on chronological values, your documents must have a numeric field containing a [UNIX timestamp](https://kb.narrative.io/what-is-unix-time). In this videogames dataset, the release year is formatted as a timestamp:
+
+```json
+[
+  {
+    "id": 0,
+    "title": "Return of the Obra Dinn",
+    "genre": "adventure",
+    "release_date": 1514761200
+  },
+  {
+    "id": 1,
+    "title": "The Excavation of Hob's Barrow",
+    "genre": "adventure",
+    "release_date": 1640991600
+  },
+  {
+    "id": 2,
+    "title": "Bayonetta 2",
+    "genre": "action",
+    "release_date": 1388530800
+  }
+]
+```
+
+If your chronological data is expressed in another format, like [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html), you must convert it to a numeric value before indexing it with Meilisearch.
+
+Most programming languages have built-in tools to help you with this process. The example below converts a game's release year, `2018`, to a numeric timestamp using JavaScript:
+
+```js
+let game = {
+  "id": 0,
+  "title": "Return of the Obra Dinn",
+  "genre": "adventure",
+  "year": 2018
+};
+
+const timestamp = Date.parse(game.year);
+
+game = {
+  "id": 0,
+  "title": "Return of the Obra Dinn",
+  "genre": "adventure",
+  "year": 2018,
+  "release_date": timestamp
+};
+```
+
+:::tip
+When preparing your dataset, it can be useful to leave the original date and time fields in your documents intact. In the example above, we keep the `year` field because it is more readable than the raw `timestamp`.
+:::
+
+After adding a numeric timestamp to all documents, index your data as usual:
+
+```sh
+curl \
+  -X POST 'http://localhost:7700/indexes/games/documents' \
+  -H 'Content-Type: application/json' \
+  --data-binary @games.json
+```
```

Comment:
This example is not related to the specific aims of this guide, and will not run for the user since there is no games.json file. My initial reaction was that it should be removed. However, I see how it can be helpful for "continuity", so to speak, since later commands are using the `games` index. So if we keep it, it needs to be clear that it is an example.

````suggestion
After adding a numeric timestamp to all documents, [index your data](/reference/api/documents.md#add-or-replace-documents) as usual. See below for an example using our nonexistent video games dataset.

```sh
curl \
  -X POST 'http://localhost:7700/indexes/games/documents' \
  -H 'Content-Type: application/json' \
  --data-binary @games.json
```
````